tests/generation/SettingsClassTest.py

```diff
@@ -0,0 +1,90 @@
+import unittest
```

Comment:
Tests should be in the same commit in which the code corresponding to them was written, so that the coverage doesn't decrease in any commit.

Reply:
👍